githubexport.py

```diff
@@ -1,88 +0,0 @@
-import requests
-import json
-# import imp
-import os
-from pathlib import Path
-from dotenv import load_dotenv
-
-load_dotenv('.env')
-# config = dotenv_values(".env")
-
-class NotFoundException(Exception):
-    def __init__(self, message, code):
-        self._message = message
-        self._code = code
-        print(self.__cause__)
-        super(message)
-
-parent = Path(__file__).parent
-file_path = parent / 'github.json'
-
-class GithubExportService:
-
-
-    url = "https://api.github.com/users/Akorede28"
-    owner = "akorede28"
-    repo = "e-commerce-API"
-    commit_sha = os.getenv('commit_sha')
-    
-
-    def get_commit_request(self,  owner, repo):
-
-        commit_url = f"https://api.github.com/repos/{owner}/{repo}/commits"
```

Comment:
Can you create a config class to hold the API URLS rather than put it in the Export services

Reply:
Class Config: 
   GITHUB_URL = "https://api.github.com"

using this 

from config import Config 

Config.GITHUB_URL